app/connectors/DestinationConnector.scala

```diff
@@ -18,18 +18,19 @@ package connectors
 
 import config.FrontendAppConfig
 import javax.inject.Inject
+import models.Movement
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.bootstrap.http.HttpClient
-import viewModels.ViewArrivalMovement
 
 import scala.concurrent.{ExecutionContext, Future}
 
+class DestinationConnector @Inject()(config: FrontendAppConfig,
+                                     http: HttpClient) {
 
-class DestinationConnector  @Inject()(config: FrontendAppConfig, http: HttpClient) {
+  def getArrivalMovements()(implicit ec: ExecutionContext,
+                            hc: HeaderCarrier): Future[Seq[Movement]] = {
 
-  def getArrivalMovements()(implicit ec: ExecutionContext, hc: HeaderCarrier): Future[ViewArrivalMovement] = {
-
-    val serviceUrl: String = s"${config.destinationUrl}/messages"
-    http.GET[ViewArrivalMovement](serviceUrl)
+    val serviceUrl: String = s"${config.destinationUrl}/arrivals-history"
```

Comment:
_arrivals-history_ Does this reflect what's being returned? Could it simply be movements? And when we need history we could use _movements/[MRN]_

Reply:
Agreed, using ArrivalsHistory kinda limits its implementation. Will make the recommended changes.